core/client/fs/src/main/java/alluxio/client/block/stream/DefaultBlockWorkerClient.java

```diff
@@ -51,58 +53,60 @@
   private static final Logger LOGGER =
       LoggerFactory.getLogger(DefaultBlockWorkerClient.class.getName());
 
-  private static final long DATA_TIMEOUT =
-      Configuration.getMs(PropertyKey.USER_NETWORK_DATA_TIMEOUT_MS);
-  private static final long KEEPALIVE_TIME_MS =
-      Configuration.getMs(PropertyKey.USER_NETWORK_KEEPALIVE_TIME_MS);
-  private static final long KEEPALIVE_TIMEOUT_MS =
-      Configuration.getMs(PropertyKey.USER_NETWORK_KEEPALIVE_TIMEOUT_MS);
-  private static final long GRPC_FLOWCONTROL_WINDOW =
-      Configuration.getBytes(PropertyKey.USER_NETWORK_FLOWCONTROL_WINDOW);
-  private static final long MAX_INBOUND_MESSAGE_SIZE =
-      Configuration.getBytes(PropertyKey.USER_NETWORK_MAX_INBOUND_MESSAGE_SIZE);
   private static final EventLoopGroup WORKER_GROUP = NettyUtils
-      .createEventLoop(NettyUtils.USER_CHANNEL_TYPE,
-          Configuration.getInt(PropertyKey.USER_NETWORK_NETTY_WORKER_THREADS),
+      .createEventLoop(
+          NettyUtils.getUserChannel(new InstancedConfiguration(ConfigurationUtils.defaults())),
```

Comment:
Well the issue is that the `WORKER_GROUP` right now is supposed to be singleton and every `DefaultBlockWorkerClient` channel should be a part of this `WORKER_GROUP`. However because it relies on configuration I had to do this messy thing to pull out the specific property. I didn't want to affect the design of this class because I didn't know what the consequences of not all channels belonging to the same configuration would be.

Reply:
Sure. I've already added the `TODO` under my name